src/pages/price/components/CraftSelectionCard/index.module.scss

```
@@ -0,0 +1,58 @@
+@import '@/styles/mixin';
```

```suggestion
@import '@/styles/mixin';
@import '@/styles/variable';
```
variableインポートしてないから、色、フォントが反映されてないじゃねと思ったが、こないだされてたよね